image_plane.go

```diff
@@ -38,43 +38,43 @@ func (p ImagePlane) index(w, h int) int {
 
 func (p ImagePlane) getValue(w, h int) float64 {
 	i := p.index(w, h)
-	if i < 0 || i >= len(p.Buffer) {
-		panic(fmt.Errorf("w %d, h %d, index %d, len(buf) %d", w, h, i, len(p.Buffer)))
+	if i < 0 || i >= len(p.Pix) {
+		panic(fmt.Errorf("w %d, h %d, index %d, len(buf) %d", w, h, i, len(p.Pix)))
 	}
-	//fmt.Printf("w %d, h %d, index %d, len(buf) %d\n", w, h, i, len(p.Buffer))
-	return p.Buffer[i]
+	//fmt.Printf("w %d, h %d, index %d, len(buf) %d\n", w, h, i, len(p.Pix))
+	return p.Pix[i]
 }
 
 func (p *ImagePlane) setValue(w, h int, v float64) {
-	p.Buffer[p.index(w, h)] = v
+	p.Pix[p.index(w, h)] = v
 }
 
 func (p *ImagePlane) getValueIndexed(i int) float64 {
-	return p.Buffer[i]
+	return p.Pix[i]
 }
 
 func (p *ImagePlane) setValueIndexed(i int, v float64) {
-	p.Buffer[i] = v
+	p.Pix[i] = v
 }
 
-func blocking(initialPlanes []*ImagePlane) ([][]*ImagePlane, int, int) {
+type Stream struct {
```

Comment:
__[gometalinter]__
* [golint] exported type Stream should have comment or be unexported